src/redux/books/bookSlice.jsx

```diff
@@ -1,25 +1,45 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  booksArr: [],
+  booksArr: [{
+    item_id: Math.floor(Math.random() * (9999 - 1000)),
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: Math.floor(Math.random() * (9999 - 1000)),
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: Math.floor(Math.random() * (9999 - 1000)),
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  }],
 };
 
 // Includes an array of books
-
+/* eslint-disable no-param-reassign */
 const bookSlice = createSlice({
```

Comment:
- [ ] Please don't disable linters because linters prevent your from preventing inconsistency and errors in your code which can break your project :fearful: Kindly remove this line and fix the error manually :+1: